dockers/docker-orchagent/orchagent.sh

```diff
@@ -18,22 +18,25 @@ fi
 mkdir -p /var/log/swss
 ORCHAGENT_ARGS="-d /var/log/swss "
 
+SWITCH_TYPE=$(echo $SWSS_VARS | jq -r '.switch_type')
 LOCALHOST_SWITCHTYPE=`sonic-db-cli CONFIG_DB hget "DEVICE_METADATA|localhost" "switch_type"`
 if [[ x"${LOCALHOST_SWITCHTYPE}" == x"chassis-packet" ]]; then
     # Set orchagent pop batch size to 128 for faster link notification handling 
     # during route-churn
     ORCHAGENT_ARGS+="-b 128 "
-else
+elif [ "$SWITCH_TYPE" != "dpu" ]; then
```

Comment:
can we make this:
```
if [[ x"${LOCALHOST_SWITCHTYPE}" == x"chassis-packet" ]]; then
elif [ "$SWITCH_TYPE" == "dpu" ]; then
else
```
It will make code more readable and will allow to extend it for other platforms as well

Reply:
Handled